projects/vdk-plugins/vdk-kerberos-auth/src/vdk/plugin/kerberos/kerberos_ticket.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+
+class KerberosTicket:
+    """
+    Adapted from http://python-notes.curiousefficiency.org/en/latest/python_kerberos.html
+    """
+
+    def __init__(self, service):
+        import kerberos
```

Comment:
Why can't we import this library at the top of the file, since both methods use it anyway?

Reply:
Fixed